### DISCUSSION TITLE
Possible New Weapon Addition - M60T

### DISCUSSION THREAD

**Discussion by SnowGar** [sh4rkman/SquadCalc#223] [Ideas]
So recently I realized that there was an experimental option that made use of the MK19 since it displays the elevation angle. The Turkish and WPMC tanks also have a feature like this, would it be possible to implement the M60's HE/HEAT round? I was about to install the SDK and mess around to see what I can find, but was wondering if there was a reason as to why it wasn't here if the MK19 RWS was.

**Comment by sh4rkman** [comment]
It's an option ! The main reason i didn't added it yet is because of the difficulty i have to test it and the imprecision it has :   

The muzzle velocity is so high that the slightest 0.1 variation in input is probably 50-100m in final distance, but i can be wrong it needs to be tested.

For the same reasons, the projectile is so fast that i can barely test it : getting out of a vehicule and jumping in an UAV takes some times and when i tested it long time ago i couldn't even see where my shots landed. Travel time is like 2-6s

**Reply by SnowGar** [reply]
I would be happy to help test if you need a hand, but the discord link on here expired at some point

**Reply by sh4rkman** [reply]
https://discord.com/invite/BNPAc5kEJP ?
It should be active, it has no expiration !

**Comment by sh4rkman** [comment]
A quick test with replacing a current weapon with the regular tank round velocity (1200m/s) and MOA (1.5) looks unreasonable : 

At this velocity and without more decimals in input, the same input can lead to >100m difference : 

![image](https://github.com/user-attachments/assets/e2a9b0e0-582f-47a7-b6bd-f1b486adc963)
